### DISCUSSION TITLE
Connection without phone - native mobile api

### DISCUSSION THREAD

**Discussion by sg957** [devlikeapro/waha#111] [Ideas]
Baileys added in the latest version the option to run the service without having to connect to WhatsApp Web, but first running the application and receiving a verification code from WhatsApp,
This has a great advantage because you do not need to have a device at the same time and the connection does not disconnect every 14 days even if the application is not open at the same time, so you can easily use virtual numbers,
Can this be added to your service?
Link to option https://github.com/WhiskeySockets/Baileys#connecting-native-mobile-api

**Comment by allburov** [comment]
Hi, it's a great feature! We'll look at this

**Comment by allburov** [comment]
I've created an issue for that, we'll look at closely! https://github.com/devlikeapro/whatsapp-http-api/issues/113

**Comment by danielsalviati** [comment]
Do you already have an estimate of when it will be implemented?
